fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -0,0 +1,80 @@
+package com.gu.facia.api
+
+import com.gu.contentapi.client.GuardianContentClient
+import com.gu.facia.api.contentapi.ContentApi
+import com.gu.facia.api.contentapi.ContentApi.{AdjustItemQuery, AdjustSearchQuery}
+import com.gu.facia.api.models._
+import com.gu.facia.client.ApiClient
+import com.gu.facia.client.models.{CollectionConfigJson, TrailMetaData}
+
+import scala.concurrent.ExecutionContext
+
+
+object FAPI {
```

Comment:
I don't mind using arguments as dependencies, but I think `implicit`s are a bit black magic, and should only really be used for typeclasses or the 'pimp my library' pattern (I wish there was a different name for that, I hate saying it!).